Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
Indeed, the second link explicitly says "The software version of your library should match the interface version.". That matches the semantic versioning contract.